tests/Action/LoginActionTest.php

```diff
@@ -310,7 +318,8 @@ private function getAction(): LoginAction
             $this->pool,
             $this->templateRegistry,
             $this->tokenStorage,
-            $this->formFactory
+            $this->formFactory,
+            $this->translator
```

Comment:
Please add a test for this

Reply:
You should test if the translator is called and that the flashbag receives the translated message

Reply:
the test for the flashbag message is above